files/docker-container-logrotate

```diff
@@ -0,0 +1,9 @@
+/var/lib/docker/containers/*/*.log {
+  rotate 0
+  hourly
+  notifempty
+  nocompress
+  size 20k
```

Comment:
Yes, I need to modify the logrotate.timer, but modifying it might have some side effects as well. I need to look into that first, which is why I am postponing this. I'll pull the change we have now and later spend time on updating logrotated's timer. `daily` log rotation should suffice our needs for now.